src/main/java/com/team4/testingsystem/entities/ContentFile.java

```diff
@@ -20,33 +22,34 @@
     @Column(name = "id")
     private Long id;
 
-    @OneToOne
-    @JsonIgnore
-    @JoinColumn(name = "question_id", nullable = false, referencedColumnName = "id")
-    private Question question;
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(
+            name = "question_content_file",
+            joinColumns = @JoinColumn(name = "content_file_id"),
+            inverseJoinColumns = @JoinColumn(name = "question_id")
+    )
+    List<Question> questions = new ArrayList<>();
 
     @Column(name = "url")
     private String url;
 
-    public ContentFile() {
-
-    }
+    public ContentFile() {}
 
-    public ContentFile(Question question, String url) {
-        this.question = question;
+    public ContentFile(String url, Question question) {
+        this.questions.add(question);
```

Comment:
Are we going to create a ContentFile for each question?

Reply:
This is used if we add a new ContentFile